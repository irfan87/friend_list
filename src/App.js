import React from "react";
import "./App.css";

function App() {
	return (
		<div className="app">
			<h1>Friends List</h1>
		</div>
	);
}

export default App;
