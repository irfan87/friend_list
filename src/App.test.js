import React from "react";
import { cleanup, render } from "@testing-library/react";
import App from "./App";

afterEach(cleanup);

it("FriendList should have snapshot", () => {
	const { asFragment } = render(<App />);

	expect(asFragment(<App />)).toMatchSnapshot();
});
