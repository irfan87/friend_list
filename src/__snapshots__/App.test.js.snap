// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendList should have snapshot 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <header>
      <nav
        class="navbar navbar-expand-lg navbar-dark bg-primary"
      >
        <div
          class="container"
        >
          <span
            class="navbar-brand"
          >
            FriendList
          </span>
        </div>
      </nav>
    </header>
    <main
      class="py-3"
    >
      <div
        class="container"
      >
        <div>
          <h1>
            Friend List
          </h1>
        </div>
      </div>
    </main>
    <footer>
      <nav
        class="navbar navbar-expand navbar-dark bg-primary"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="text-center py-3 col"
            >
              Copyright Â© FriendList
            </div>
          </div>
        </div>
      </nav>
    </footer>
  </div>
</DocumentFragment>
`;
